staging/src/k8s.io/kube-aggregator/pkg/client/listers/apiregistration/v1/apiservice.go

```diff
@@ -39,30 +39,10 @@ type APIServiceLister interface {
 
 // aPIServiceLister implements the APIServiceLister interface.
 type aPIServiceLister struct {
```

Comment:
LOL at aPIService.  Can we file a bug to fix the namer to recognize "API" -> "api"